### ISSUE TITLE
Generate debug strings

### ISSUE THREAD

**Issue by ZevEisenberg** [ZevEisenberg/manuscript#1]
It can be difficult to debug advanced attributed strings, because certain characters (image attachments, various kinds of white space or zero-width characters, Unicode character) are hard to decipher in `-[NSAttributeString description]`’s output. Manuscript should be able to output debug strings like this (features labeled for emphasis):

```
                                                    ↓ no need to call out normal inner spaces
[24×36 px attached image][non-breaking thin space]My Super Duper String[Trailing Space]
 ↑ image attachments      ↑ proper Unicode names of whitespace chars    ↑ hard-to-see characters
```

It should also be able to generate an Objective-C (or Swift?) line of code that would use a format string to recreate this string, for use in unit tests or for debugging purposes.

**Comment by ZevEisenberg** [comment]
It would probably also be useful to (optionally) indicate attribute changes that would otherwise not appear in the attributed string’s `.string`:

```
A string with [attribute change]bold [attribute change] and [attribute change]italic [attribute change] text
```

**Comment by ZevEisenberg** [comment]
Fixed in cdb55dcd6dc2f591ae099c8e741eee3ec8204c5e (without nice-to-have attribute changes)
